packages/snaps-controllers/src/interface/utils.ts

```diff
@@ -59,7 +80,12 @@ function constructInputState(
   oldState: InterfaceState,
   element: InputElement | DropdownElement,
 ) {
-  return element.props.value ?? oldState[element.props.name] ?? null;
+  return (
+    element.props.value ??
+    oldState[element.props.name] ??
+    constructComponentSpecificDefaultState(element) ??
+    null
```

Comment:
`constructComponentSpecificDefaultState` already returns null at the end, so we need the last null here ?

Reply:
Hmm, I was thinking if `constructComponentSpecificDefaultState` would return undefined in the future it might be nice to keep the null as a fallback 🤔

Reply:
We can keep it as a fallback, it's a good argument.